src/components/Book.js

```diff
@@ -16,7 +16,7 @@ const Book = ({
         <h2 className="h2-title">{title}</h2>
         <span className="author">{author}</span>
         <button type="button" className="button">Comments</button>
-        <button type="button" className="button-active" onClick={() => { dispatch(remove(id)); }}>Remove</button>
+        <button type="button" className="button-active" onClick={() => { dispatch(removeBook(id)); }}>Remove</button>
```

Comment:
- [ ] I am unable to verify if a book is removed from API because a book cannot be added in the first place. Kindly fix that and ensure books can be removed from the API, not just the store.